actors/miner/src/lib.rs

```diff
@@ -4351,10 +4368,11 @@ fn get_verify_info(
 
     let commd = unsealed_cid.get_cid(params.registered_seal_proof)?;
 
+    // Maybe sealverifyinfo is where new cspv things go? its maybe more for actual proof verification
```

Comment:
I think SVI is coupled to proof verification, and should probably stay limited to that. I think the presence of deal IDs is a bug in the FVM/proofs API.